dvclive/lightning.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, Optional, Union
+
+from pytorch_lightning.loggers import LightningLoggerBase
+from pytorch_lightning.loggers.base import rank_zero_experiment
+from pytorch_lightning.utilities import rank_zero_only
+from torch import is_tensor
+
+import dvclive
+
+
+class DvcLiveLogger(LightningLoggerBase):
+    def __init__(
+        self,
+        run_name: Union[str, int],
+        save_dir: Optional[str] = None,
+        summary: Optional[bool] = True,
+        prefix="",
+        experiment=None,
```

Comment:
In order to match other integrations are going to do (#167) I would just use `**kwargs` for `dvclive` args :

```suggestion
        prefix="",
        experiment=None,
        **kwargs
```